bootstrap/eks/config/manager/manager.yaml

```
@@ -26,3 +26,10 @@ spec:
       tolerations:
         - effect: NoSchedule
           key: node-role.kubernetes.io/master
+      affinity:
+       nodeAffinity:
+         requiredDuringSchedulingIgnoredDuringExecution:
+           nodeSelectorTerms:
+           - matchExpressions:
+             - key: node-role.kubernetes.io/master
+               operator: Exists
```

As `node-role.kubernetes.io/master` label is going away should we take this opportunity to include the new `node-role.kubernetes.io/control-plane` label to support future versions of Kubernetes (while keeping the old label for now)
```suggestion
        - effect: NoSchedule
          key: node-role.kubernetes.io/control-plane
        - effect: NoSchedule
          key: node-role.kubernetes.io/master
      affinity:
       nodeAffinity:
         requiredDuringSchedulingIgnoredDuringExecution:
           nodeSelectorTerms:
           - matchExpressions:
             - key: node-role.kubernetes.io/control-plane
               operator: Exists
           # remove once usage of node-role.kubernetes.io/master is removed from Kubernetes           
           - matchExpressions:           
             - key: node-role.kubernetes.io/master
               operator: Exists
```